test/test_rtfcre_dict.py

```diff
@@ -31,8 +31,8 @@ def test_converter(self):
         (r'\}', '}'),
         (r'\~', '{^ ^}'),
         (r'\_', '-'),
-        ('\\\r', '{#Return}{#Return}'),
-        ('\\\n', '{#Return}{#Return}'),
```

Comment:
This covered the case of a line ending with a backslash, I'm not sure if this is in the RTF CRE spec. If it is, we can add this functionality back.